package.json

```diff
@@ -11,7 +11,7 @@
     "eslint": "eslint src stories .storybook",
     "test:ci": "yarn test && yarn lint",
     "deploy-storybook": "storybook-to-ghpages",
-    "build": "babel src -d dist"
+    "build": "rm -rf dist/* && babel src -d dist --ignore '**/*.test.js'"
```

Comment:
Can the ignore be moved to babelrc?

Reply:
```
"ignore": [
  "**/__tests__", 
  "**/*.test.js"
]
```
does the above work?

Reply:
I don't believe so. If we move it to the `babelrc` then when we run the tests, they are not run through babel which causes them to fail since we need transpiling to run.

Reply:
To clarify, I tried adding something similar to the above, but it causes the test suite to fail.